siliconcompiler/tools/openroad/openroad.py

```diff
@@ -345,7 +345,7 @@ def post_process(chip):
                     drvs += int(metrics[metric])
 
         if drvs is not None:
-            chip.set('metric', step, index, 'drvs', str(drvs), clobber=True)
+            chip.set('metric', 'drvs', str(drvs), clobber=True, step=step, index=index)
```

Comment:
just for readability, it might be good to swap step/index and clobber

Reply:
Nice catch, ended up getting rid of the `clobber=True` on the metrics from here and in Yosys driver since it's unnecessary (default is True, plus metrics shouldn't be set by anything else-don't think the explicit setting communicates much).